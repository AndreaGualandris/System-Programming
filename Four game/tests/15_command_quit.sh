#!/bin/bash
exec "$1" 17 7 < tests/15_command_quit.txt
