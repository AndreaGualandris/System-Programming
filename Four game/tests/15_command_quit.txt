5
2
5
0
4
6
6
2
2
1
3
3
1
6
0
1
4
5
2
6
q
7
8
9
10
