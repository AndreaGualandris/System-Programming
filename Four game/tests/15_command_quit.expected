-----------------
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
     o           
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
  x  o           
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
     o           
  x  o           
-----------------
01234567890123456
Player 'x' to move:
x
v
-----------------
                 
                 
                 
                 
                 
     o           
x x  o           
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
     o           
x x oo           
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
     o           
x x oox          
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
     oo          
x x oox          
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
  x  oo          
x x oox          
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
                 
  o              
  x  oo          
x x oox          
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
  o              
  x  oo          
xxx oox          
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
  o              
  x  oo          
xxxooox          
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
  o              
  xx oo          
xxxooox          
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
  o              
 oxx oo          
xxxooox          
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
  o   x          
 oxx oo          
xxxooox          
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
  o   x          
ooxx oo          
xxxooox          
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
 xo   x          
ooxx oo          
xxxooox          
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
 xo   x          
ooxxooo          
xxxooox          
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
 xo  xx          
ooxxooo          
xxxooox          
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
  o              
 xo  xx          
ooxxooo          
xxxooox          
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
  o   x          
 xo  xx          
ooxxooo          
xxxooox          
-----------------
01234567890123456
Player 'o' to move:
